openedx/core/lib/tests/test_grade_utils.py

```diff
@@ -45,3 +45,8 @@ def test_compare_scores_treat_undefined_as_zero(self):
         assert is_higher is True
         assert 0 == percentage_1
         assert 0 == percentage_2
+
+    def test_round_away_from_zero(self):
```

Comment:
This is a good candidate for ddt, and you should include the case that produced -0, plus a bunch of others, like zero, positive zero, etc.